internal/gengapic/paging.go

```diff
@@ -389,3 +389,20 @@ func (g *generator) pagingIter(pt *iterType) {
 		g.imports[spec] = true
 	}
 }
+
+func (g *generator) pagingIterGo123(pt *iterType) {
+	p := g.printf
+
+	p("// All returns an iterator. If an error is returned by the iterator, the")
+	p("// iterator will stop after that iteration.")
```

Comment:
Maybe mention here how to access the error?

Reply:
I decided to just generate an example of the usage. LMK if that is good enough or you still think we should say more.

Reply:
I think the example is great, it makes everything clear.